### ISSUE TITLE
When we end conference call that time we get any callback?

### ISSUE THREAD

**Issue by Ashish1112** [voxeet/voxeet-ios-sdk#13]
When we end conference call that time we get any callback so i can call function and perform one functionality

Thanks in advance

**Comment by VTCoco** [comment]
What? Can you rephrase please?

**Comment by Ashish1112** [comment]
Please look at below Screenshot

https://user-images.githubusercontent.com/21678840/65311229-91981880-dbad-11e9-9f32-43a32c45b9e5.png

When we click on that red marked button then any event or any callback is there to identify call is ended and we call some function on it?

Thanks

**Comment by VTCoco** [comment]
You can use some notification like them: https://github.com/voxeet/voxeet-ios-sdk#handling-notifications

For example if you want to know when **everybody** left the conference observe "**VTConferenceDestroyedPush**".
If you are using CallKit you can observe "**VTCallKitEnded**" that will tell you when the conference ends.

**Comment by Ashish1112** [comment]
Thank you so much for your support..!
